server/models/Notification.js

```diff
@@ -0,0 +1,40 @@
+const mongoose = require("mongoose");
+
+const notificationSchema = new mongoose.Schema({
+    noti_type: {
```

Comment:
Not a big deal but should probably spell this out completely for readability sake